routing/matcher_test.go

```diff
@@ -237,6 +270,12 @@ func initRandomPaths() {
 	benchmarkMatchersInitialzed = true
 }
 
+func initBenchmark(b *testing.B, init func()) {
+	b.StopTimer()
+	init()
+	b.StartTimer()
```

Comment:
Can't you replace the Stop/Start timer with just a ResetTimer after init() ? Better yet, if there is no init (see above), it shouldn't affect the benchmarks